qa/rpc-tests/wallet_shieldcoinbase_sapling.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python2
+import inspect
+import os
+
+cwd = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
+execfile(os.path.join(cwd, 'wallet_shieldcoinbase.py'))
```

Comment:
What are the above lines doing?

Reply:
The way the RPC test runner works, the various RPC test files don't actually see themselves as being in the same module. This code finds the file we want in the same directory, and then executes it as if it was code in the same file.